scanhub-ui/src/components/Navigation.tsx

```diff
@@ -33,45 +36,59 @@ import LoginContext from '../LoginContext'
 // import { SettingsInputSvideoRounded } from '@mui/icons-material';
 import { loginApi } from '../api'
 import { UserRole } from '../generated-client/userlogin'
+import ScanhubLogo from '../media/ScanhubLogo.png'
+import NotificationContext from '../NotificationContext'
+
 
 function ColorSchemeToggle() {
   const { mode, setMode } = useColorScheme()
   const { setMode: setMuiMode } = useMaterialColorScheme()
   const [mounted, setMounted] = React.useState(true)
-  React.useEffect(() => {
-    setMounted(true)
+
+  React.useEffect(() => {  // this effect is somehow needed for server side rendering according to the docs.
+    setMounted(true)       // probably not needed in our case.
   }, [])
 
+
   if (!mounted) {
     return <IconButton size='sm' variant='outlined' color='primary' />
   }
 
+  let modeicon = <BrightnessAutoIcon />
```

Comment:
See line 65, the icon of the mode the user can switch to should be displayed.

Reply:
Possible and easily changeable, but with the new automatic option, I find that confusing too.

Reply:
True, then lets just keep it as it is.